easybuild/easyconfigs/b/Brunsli/Brunsli-0.1-GCCcore-13.2.0.eb

```diff
@@ -13,7 +13,12 @@ toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
 
 source_urls = ['https://github.com/google/brunsli/archive/']
 sources = ['v%(version)s.tar.gz']
-checksums = ['62762dc740f9fcc9706449c078f12c2a366416486d2882be50a9f201f99ac0bc']
+patches = ['Brunsli-0.1-fix-Brotli-static-library-names.patch']
+checksums = [
+    {'v0.1.tar.gz': '62762dc740f9fcc9706449c078f12c2a366416486d2882be50a9f201f99ac0bc'},
+    {'Brunsli-0.1-fix-Brotli-static-library-names.patch': \
+         '0b921fb4839f0b0223c1bf08499e84afd5d428f448664fd1337173f45a815704'},
```

Comment:
```suggestion
    {'Brunsli-0.1-fix-Brotli-static-library-names.patch': \
     '0b921fb4839f0b0223c1bf08499e84afd5d428f448664fd1337173f45a815704'},
```

Reply:
> E127 continuation line over-indented for visual indent

basically means: You indented to much to match the visual indentation (as opposed to logical)

Inside braces/parens you don't need the newline-escape.     
Hence you just need to make it match the previous indent. That could be the open-brace but as we are still *inside* we need an extra space

You can use `autopep8` to automatically format the file and/or integrate it into your editor/IDE